types/types.go

```diff
@@ -354,12 +356,12 @@ type ThrottleDevice struct {
 // ShellCommand is a string or list of string args.
 //
 // When marshaled to YAML, nil command fields will be omitted if `omitempty`
-// is specified as a struct tag. Explicitly empty commands (i.e. `[]` or `''`)
+// is specified as a struct tag. Explicitly empty commands (i.e. `[]` or `”`)
```

Comment:
@ndeloof you're IDE again?
```suggestion
// is specified as a struct tag. Explicitly empty commands (i.e. `[]` or `''`)
```

Reply:
😓 I thought I found the right option to prevent this.